spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/spring-cloud-gcp-pubsub-bus-config-sample-server-github/pom.xml

```diff
@@ -11,7 +11,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>spring-cloud-gcp-pubsub-bus-config-sample-server-github</artifactId>
-  <name>Spring Cloud GCP Pub/Sub Bus Sample (GitHub Configuration)</name>
+  <name>Spring Cloud GCP Code Sample - Pub/Sub Bus (GitHub Configuration)</name>
```

Comment:
I probably misnamed this one in the first place. It's an example of using Spring Cloud Config with underlying messaging provided by Spring Cloud Bus powered by GCP Cloud Pub/Sub. A mouthful...

Maybe `Spring Cloud GCP Code Sample - Spring Cloud Config with Pub/Sub Bus (GitHub Configuration)`?

Reply:
Done.